README.rst

```diff
@@ -214,26 +214,32 @@ Usage from Docker
 
 By default, docker will not copy the ``.git``  folder into your container.
 Therefore, builds with version inference might fail.
-Consequently, you can use the following snipped to infer the version from the host os without copying the entire ``.git`` folder to your Dockerfile.
+Consequently, you can use the following snipped to infer the version from
+the host os without copying the entire ``.git`` folder to your Dockerfile.
 
 .. code:: dockerfile
 
     RUN --mount=source=.git,target=.git,type=bind \
         pip install --no-cache-dir -e .
 
 However, this build step introduces a dependency to the state of your local
-.git folder the build cache.
-To optimize build caching, one can use ``SETUPTOOLS_SCM_PRETEND_VERSION=1`` in
-previous build step, so that a full docker-file might look like that:
+.git folder the build cache and triggers the long-running pip install process on every build.
+To optimize build caching, one can use an environment variable to pretend a pseudo
+version that is used to cache the results of the pip install process:
 
 .. code:: dockerfile
 
     FROM python
     COPY pyproject.toml
-    RUN SETUPTOOLS_SCM_PRETEND_VERSION=1 pip install -e .[test]
+    RUN SETUPTOOLS_SCM_PRETEND_VERSION=ignore pip install -e .[test]
```

Comment:
this versions will break certain cyclic dependencies where a specfic/working/valid version for a dependency is actually needed

Reply:
oups. fixed.